.circleci/config.yml

```diff
@@ -222,6 +222,7 @@ jobs:
       - run: make dialyzer
       - run: make swagger-check
       - run: make rebar-lock-check
+      - run: ./rebar3 xref || true
```

Comment:
Make target would be preferable.

Reply:
To be honest I really don't like the current state (abuse) of the Makefile.
Make is build tool, not task runner.